05-dom-document-loading/2-sortable-table-v1/index.js

```diff
@@ -1,6 +1,89 @@
 export default class SortableTable {
   constructor(headerConfig = [], data = []) {
+    this.headerConfig = headerConfig;
+    this.data = data;
+    this.render();
+  }
+
+  render() {
+    this.element = this.createElement(this.createTemplate());
+    this.subElements = this.getSubElements(this.element);
+  }
```

Comment:
constructor(headerConfig = [], data = []) {
    this.headerConfig = headerConfig;
    this.data = data;
    this.element = this.createElement(this.createTemplate());
    this.subElements = this.getSubElements(this.element);
  }